hyph-utf8/tex/generic/hyph-utf8/patterns/tex/hyph-lv.tex

```diff
@@ -1,16 +1,25 @@
-% This file is part if hyph-utf8 package and resulted from semi-manual
-% conversions of hyphenation patterns into UTF-8 in November 2008 by Mojca &
-% Arthur.
+% title: Hyphenation patterns for Latvian
+% copyright: Copyright (C) 2004-2005 Janis Vilims, jvilims@apollo.lv
+% notice: This file is part of the hyph-utf8 package.
+%     See http://www.hyphenation.org for more information
+%     Original source Latvian Lingucomponents for OpenOffice,
+%     http://sourceforge.net/projects/openoffice-lv,
+%     file hyph_lv_lv.dic (version 0.3, 2005-09-14)
```

Comment:
I'm surprised that we don't have a special field for source.

Reply:
We do now :-)